backend/internal/pkg/repositories/auth/auth.go

```diff
@@ -0,0 +1,31 @@
+package auth
+
+import (
+	"context"
+	"errors"
+
+	"github.com/ParkWithEase/parkeasy/backend/internal/pkg/models"
+	"github.com/google/uuid"
+)
+
+var (
+	ErrIdentityNotFound  = errors.New("requested identity not found")
+	ErrDuplicateIdentity = errors.New("an identity already exist")
+)
+
+type Identity struct {
+	Email        string
+	PasswordHash models.HashedPassword
+	Id           uuid.UUID
+}
+
+type Repository interface {
```

Comment:
kinda nit pit but I suggest changing Repository to IRepository so it is clear this is an interface

Reply:
Lol